Makefile

```diff
@@ -221,3 +221,12 @@ run-on-kind:
 	$(CMD_PREFIX) kubectl create namespace instructlab-bot
 	$(CMD_PREFIX) kubectl create -n instructlab-bot secret generic instructlab-bot --from-env-file=.env
 	$(CMD_PREFIX) kubectl apply -k deploy/instructlab-bot/overlays/dev
+
+.PHONY: docker-clean
```

Comment:
Can we change this to container-clean. Prolly best to avoid docker refs with the licensing issues around Docker Desktop. Ty!